pkg/admission/customresourcevalidation/securitycontextconstraints/validation/validation.go

```diff
@@ -0,0 +1,350 @@
+package validation
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/validation"
+	apimachineryvalidation "k8s.io/apimachinery/pkg/api/validation"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/kubernetes/pkg/apis/core"
```

Comment:
Could you name it `kcore` and the one below `kcorev1` or anything that comes to your mind in order to distinguish this from the external `corev1` above?

Reply:
in changing that i fully realized why :|  fixed